react/features/mobile/call-integration/middleware.js

```diff
@@ -228,14 +228,16 @@ function _conferenceWillJoin({ dispatch, getState }, next, action) {
     const state = getState();
     const { callHandle, callUUID } = state['features/base/config'];
     const url = getInviteURL(state);
-    const handle = callHandle || url.toString();
```

Comment:
same as above, this change is incorrect: a conference will always have a URL

Reply:
This is not my change

Reply:
How so? You care modyfying this thing now.